guides/common/modules/proc_importing-ansible-variables-from-yaml.adoc

```diff
@@ -0,0 +1,16 @@
+[id="importing-Ansible-variables-from-yaml-files-{context}"]
+= Importing Ansible variables from YAML files
+
+You can import Ansible variables from a `.yaml` file to {Project} within the {ProjectWebUI}.
+
+.Prerequisites
+* You must have at least one Ansible role imported.
+For more information, see xref:Importing_Ansible_Roles_and_Variables_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Ansible* > *Variables*.
+. Click *Import from YAML-File* in the toolbar.
++
+A wizard opens to show you the next steps to import and override Ansible variables.
```

Comment:
This line basically tells the user to just "follow the wizard". This was done in light of #2490.
Since my wizard also contains helpertexts and is laid out in a generally self-explanatory fashion, I found it fitting.

For the wording, I oriented myself towards:
https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_standardizing-content-view-empty-states.adoc?plain=1#L6